<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStatusWnd</class>
 <widget class="QDockWidget" name="CStatusWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>175</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Status</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QTabWidget" name="statusSheet">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="analysisPage">
       <attribute name="title">
        <string>Player Analyses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="playAnalysisShowWest">
              <property name="text">
               <string>West</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="playAnalysisShowNorth">
              <property name="text">
               <string>North</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="playAnalysisShowEast">
              <property name="text">
               <string>East</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="playAnalysisShowSouth">
              <property name="text">
               <string>South</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="analysisWest" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>West</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="playerAnalysisWest">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="plainText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="analysisNorth" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>North</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="playerAnalysisNorth">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="analysisEast" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>East</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="playerAnalysisEast">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="analysisSouth" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>South</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="playerAnaslysisSouth">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="playPlanPage">
       <attribute name="title">
        <string>Play Plan</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Declarer:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="playPlanDeclarer">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string># Plays:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="playPlanPlays">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Current:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="playPlanCurrent">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="playPlan">
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>12</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cardLocationsPage">
       <attribute name="title">
        <string>Card Locations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Observer:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cardLocationsObserver"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Target:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cardLocationsTarget"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="cardLocations">
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>6</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="holdingsPage">
       <attribute name="title">
        <string>Holdings</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Combined Suit Status</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="holdings">
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
